dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -422,13 +423,14 @@ public Workflow refresh(@ApiParam(hidden = true) @Parameter(hidden = true, name
             @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Workflow.class)
     public Workflow refreshVersion(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User user,
             @ApiParam(value = "workflow ID", required = true) @PathParam("workflowId") Long workflowId,
-            @ApiParam(value = "version", required = true) @PathParam("version") String version) {
+            @ApiParam(value = "version", required = true) @PathParam("version") String version,
+            @ApiParam(value = "refresh all versions", defaultValue = "true") @QueryParam("hardRefresh") @DefaultValue("true") Boolean hardRefresh) {
```

Comment:
Agreed with @garyluu 's comment down below in generated YAML -- this is saying do a hard refresh of this particular version, not all versions.

Reply:
Pretty sure this is the case, since it's an optional query parameter, but just to confirm, the CLI will continue to work against the old enpoint, right?

Reply:
The CLI should work with the old endpoint since the value is optional and defaults to true (equivalent to current refresh)

Reply:
Subscribe to and/or this issue which would make the test code at least a lot cleaner
https://github.com/swagger-api/swagger-codegen/issues/8939

Reply:
Yeah I saw that too, fingers crossed 🤞